app/views/articles/show.html.erb

```diff
@@ -1,2 +1,77 @@
-<h1>Articles#show</h1>
-<p>Find me in app/views/articles/show.html.erb</p>
+<div class="container">
+  <div class="row">
+
+    <div class="col-xs-8">
+
+      <h2><%= @article.title %></h2>
+      <p><em><%= @article.summary %></em></p>
+      <p>(£ <%= @article.price %> per day) </p>
+
+      <!-- Show URL/content only if accessible, i.e. owned or booked -->
+
+      <% accessible_articles = current_user.articles + current_user.today_booked_articles %>
+
+      <% if accessible_articles.include? @article %>
+        <%= link_to @article.url, @article.url %>
+      <% else %>
+        <p>You do not have access to this article</p>
+      <% end %>
```

Comment:
Displays URL (or whatever is purchased through the booking) if it is currently booked. Otherwise we notify the user they do not have access, and they will have on the RHS a CTA (booking form)